src/panels/config/logs/system-log-card.ts

```diff
@@ -31,10 +33,41 @@ export class SystemLogCard extends LitElement {
     this._items = await fetchSystemLog(this.hass!);
   }
 
+  private _timestamp(item: LoggedError): string {
+    return formatSystemLogTime(item.timestamp, this.hass!.locale);
+  }
+
+  private _multipleMessages(item: LoggedError): string {
+    return this.hass.localize(
+      "ui.panel.config.logs.multiple_messages",
+      "time",
+      formatSystemLogTime(item.first_occurred, this.hass!.locale),
+      "counter",
+      item.count
+    );
+  }
+
   protected render(): TemplateResult {
     const integrations = this._items
       ? this._items.map((item) => getLoggedErrorIntegration(item))
       : [];
+    const filteredItems = this._items
```

Comment:
Use `memoizeOne` so we only calculate this if `_filter` changes

Reply:
Thanks, done